src/scss/components/_login-button-logo.scss

```diff
@@ -1,17 +1,37 @@
+@use '~microscope-sass/lib/bem';
+
 @import '~microscope-sass/lib/grid';
 @import '~microscope-sass/lib/responsive';
 
 @import '../mixins/prefix';
 
 .#{prefix(login-button-logo)} {
-  display: flex;
+  display: inline-flex;
   align-items: center;
 
+  &:focus,
+  &:focus-visible {
+    border-color: var(--_of-login-button-logo-focus-color-border);
+    border-width: 2px;
+    outline: var(--_of-login-button-logo-focus-color-border) solid 2px;
+    outline-offset: 1px;
+    border-radius: var(--of-login-button-focus-border-radius, 1px);
+  }
+
   @include mobile-only {
     width: 25%;
     justify-content: center;
   }
 
+  @include bem.modifier('dark') {
+    --_of-login-button-logo-focus-color-border: var(--of-login-button-logo-dark-focus-color-border);
+  }
+  @include bem.modifier('light') {
+    --_of-login-button-logo-focus-color-border: var(
+      --of-login-button-logo-light-focus-color-border
+    );
+  }
```

Comment:
we prefer using the `bem` library for specific CSS selectors. Also, we specify only the necessary overrides rather than duplicating code.